homeassistant/components/open_meteo/weather.py

```diff
@@ -121,3 +135,7 @@ def forecast(self) -> list[Forecast] | None:
             forecasts.append(forecast)
 
         return forecasts
+
+    async def async_forecast_daily(self) -> list[Forecast] | None:
+        """Return the daily forecast in native units."""
+        return self.forecast
```

Comment:
Should this be a default in the parent class? Feels odd to implement this everywhere.

Reply:
No, because:
- The `forecast` property is deprecated and will go away soon
- In most cases it's now known if the `forecast` property will return the daily forecast or some other forecast